test/task-store.spec.coffee

```diff
@@ -15,13 +15,13 @@ describe 'Task Store', ->
     expect(TaskStore.isUnknown(id)).to.be.true
 
 
-  it 'should load a task and notify', ->
+  it 'should load a task and notify', (done)->
     calledSynchronously = false
     TaskStore.addChangeListener ->
       calledSynchronously = true
+      calledSynchronously && done()
```

Comment:
small nitpick; could you use `and`, `or`, '`is', `isnt` instead of `&&`, `||`, `===`, `!==`? It's mostly useful for a consistent `===` and `!==`.